src/app/design/guidelines/managing-records/index.html

```diff
@@ -28,15 +28,12 @@
   </div>
 
   <div class="sky-subsection-heading" style="margin-top: 20px;">
-    Add from a search field
+    Add from a search
   </div>
 
   <p>
-    A record may need to reference another record, such as a donation that references a constituent. If the referenced record does not exist, the user needs to add it on the fly. The user should search first to avoid duplicate records. After confirming that the record to reference does not exist, the user can invoke the add form.
+    A record may need to reference another record, such as a donation that references a constituent. If the referenced record does not exist, the user needs to add it on the fly. The user should search first to avoid duplicate records. After confirming that the record to reference does not exist, the user can invoke the add form. The <a href="https://developer.blackbaud.com/skyux-lookup/docs/lookup">lookup</a> component has an option to include a button to invoke an add form.
```

Comment:
Could be more prescriptive here, along the lines of "Use the lookup component if users need to add items during a search"

Reply:
```suggestion
    A record may need to reference another record, such as a donation that references a constituent. If the referenced record does not exist, the user needs to add it on the fly. The user should search first to avoid duplicate records. After confirming that the record to reference does not exist, the user can invoke the add form. To let the user invoke an add form during a search, use the <a href="https://developer.blackbaud.com/skyux-lookup/docs/lookup">lookup</a> component.
```
Here's an edit to make it more prescriptive as Todd suggested. I used `user` singular even though we usually make this plural because it's singular throughout this page. ... If we don't want to make this more prescriptive, I think the current text is fine.

Reply:
I like that better.